src/browser.js

```diff
@@ -6,6 +6,9 @@ import Modal from './components/Modal';
 import toggleClass from './toggle-class';
 import getScrollbarSize from './utils/scrollbarSize';
 
+// This keeps track of how many modals have been created so that the
+// container padding for the scrollbar is correctly set.
+let numberOfModalsCreated = 0;
```

Comment:
Why are you tracking created modals? There is no limit on how many modals can be created in tg-modal. The current limitation is only for opened modals.